crates/ignore/src/default_types.rs

```diff
@@ -52,6 +52,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("cshtml", &["*.cshtml"]),
     ("css", &["*.css", "*.scss"]),
     ("csv", &["*.csv"]),
+    ("dts", &["*.dts", "*.dtsi"]),
```

Comment:
I don't think `dts` is widely recognized enough to be its own type name. Could you rename this to `devicetree` please? Also, please follow the conventions of the surrounding (and the comment above) and make sure the list is sorted lexicographically.

Reply:
Done. Not sure how that mis-sorting happened, but that's fixed too now.